package.json

```diff
@@ -41,7 +34,8 @@
     "release": "semantic-release",
     "prerelease": "semantic-release --dry-run",
     "dependencies:check": "ncu",
-    "tools-setup": "echo 'nothing to do for now'"
+    "tools-setup": "echo 'nothing to do for now'",
+    "prepare": "husky install"
```

Comment:
I'm fairly sure for v9 of husky, we should only need to do `"prepare": "husky"`. `husky install` is now deprecated.